src/system/EventPipe.hxx

```diff
@@ -17,7 +17,7 @@
  */
 class EventPipe {
 #ifdef _WIN32
-	UniqueSocketDescriptor r, w;
+	SocketDescriptor r, w;
```

Comment:
No, `UniqueSocketDescriptor` is correct; because who will close the socket?
The `UniqueSocketDescriptor` was just missing in mpdscribble; this class was copied from MPD, but since nobody tried building mpdscribble on Windows, this missing source file went unnoticed.